scripts/dist.js

```diff
@@ -31,6 +31,9 @@ function getVersion (type, done) {
 function writeData (type, version, done) {
   const dataPath = join(RELEASE_PATH, type, version, 'dist.json')
   const data = JSON.parse(fs.readFileSync(dataPath).toString())
+  if (!data.dateUTC && data.date) {
+    data.dateUTC = new Date(data.date).toUTCString()
```

Comment:
I'd prefer to do this in the template rather than sneak it in here, but it's not a huge issue either way.

Reply:
Me too. I initially tried to use the [`dateFormat`](https://gohugo.io/functions/dateformat/) function provided by Hugo, but it cannot parse the date format used in the `data.json` files. 

Another option would be to change the date format used in the `data.json` files so they can be formatted as needed using the `dateFormat` function.

Reply:
Ah... and we can't define a custom function. Eh, nevermind.